README.md

```diff
@@ -24,9 +24,9 @@ bootstrap({
 
  * Creates and starts the server listening for connections.
  * `@param {Object}` options
- * `@return {Object} bootstrap` interface object.
+ * `@return {Promise.<bootstrap, Error>}` a promise that returns bootstrap interface object if resolved, Error if rejected.
```

Comment:
That conversation never really concludes, but examples include syntax with a pipe, which I prefer to comma because it's an either or scenario and comma used in other contexts express multiple arguments not binary. `@return {Promise.<bootstrap|Error>}`

Does that make sense, how do you feel about this?